includes/class-boldgrid-editor-assets.php

```diff
@@ -163,6 +163,31 @@ public static function is_webpack() {
 		return ( defined( 'BGEDITOR_SCRIPT_DEBUG' ) && BGEDITOR_SCRIPT_DEBUG );
 	}
 
+	/**
+	 * Maybe Enqueue Animate.css
+	 *
+	 * Checks if post uses the animate.css or not.
+	 *
+	 * @since SINCEVERSION
+	 *
+	 * @return bool True if animate.css is to be enqueued.
+	 */
+	public function maybe_enqueue_animate() {
+		global $post;
+
+		if ( ! $post ) {
+			return true;
```

Comment:
Should we return false if there is no post?

Reply:
No.... Because I want the default behavior to be to load the animate.css, unless we specifically make the determination that the content lacks the required classes.

This will avoid potential issues of the animate.css NOT loading when it SHOULD load.